src/view/upcastwriter.js

```diff
@@ -17,6 +19,49 @@ import { isPlainObject } from 'lodash-es';
  * see {@link module:engine/view/downcastwriter~DowncastWriter writer}.
  */
 export default class UpcastWriter {
+	/**
+	 * Creates new {@link module:engine/view/documentfragment~DocumentFragment}.
```

Comment:
```suggestion
	 * Creates a new {@link module:engine/view/documentfragment~DocumentFragment}.
```

We don't need to specify the link's text in this case because we'd just repeat the default one.